### PR TITLE
ssr: Extract Card Space loading

### PR DESCRIPTION
**Pull request by backspace**
This extracts a Card Space resource and a service that uses it and changes
the index route to load the model via the service. This is a step toward editing
routes nested under `/`. I tried out that nesting and had some problems, will
have to look separately.

### CODE REVIEW

**Review by Aierie — APPROVED** [review]

**Comment by Aierie** [comment]
>  I tried out that nesting and had some problems, will have to look separately.

What problems did you have w the nesting?

File: packages/ssr-web/app/resources/card-space.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    super(owner, args);
+  }
+
+  async run() {
+    try {
+      await this.fetchCardSpace(this.args.named.slug);
+      this.loading = false;
+    } catch (err) {
+      this.errored = err;
+      this.loading = false;
```
[the remark this answers is not in the document]
Comment (backspace) [reply packages/ssr-web/app/resources/card-space.ts]: makes sense, done in 9084892, thanks

File: packages/ssr-web/app/services/card-space.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  @service declare layer2Network: Layer2Network;
+
+  model = useResource(this, CardSpace, () => ({
+    named: {
+      slug: this.slug,
+    },
+  }));
+
+  get isActive() {
+    return this.appContext.isCardSpace;
```
[the remark this answers is not in the document]
Comment (backspace) [reply packages/ssr-web/app/services/card-space.ts]: I’ve now arranged things so most relevant properties are in the Card Space service… maybe dramatique enough that this is worth another look 🙃

**Comment by backspace** [comment]
> What problems did you have w the nesting?

It caused the tests to fail on pay and `/wc`… hopefully there’s a way forward but it felt like too much for this PR.

File: packages/ssr-web/app/services/card-space.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (backspace) [reply packages/ssr-web/app/services/card-space.ts]: okay, I removed that rearrangement and succeeded at fixing the route nesting. The Card Space model loads in `application`, which is somewhat unorthodox, but allows `/edit` to inherit the model. There’s a placeholder edit template:
image

I think this is a good place to wrap up this PR so edit UI can proceed 🎉

[the remark this answers is not in the document]
Comment (Aierie) [reply packages/ssr-web/app/services/card-space.ts]: Ya, I agree! We can try out adding editing and we'll know whether stuff fits nicely

**Pull request merged**